src/components/Banner.tsx

```diff
@@ -0,0 +1,48 @@
+import { Card, CardBody, Center, Heading, Image, LinkOverlay, Stack } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+
+import { RawText } from './Text'
+
+const display = { base: 'none', md: 'flex' }
+
+type BannerProps = {
+  maxWidth?: number;
+  iconSrc: string;
+  titleKey: string;
+  descriptionKey: string;
+  ctaKey: string;
+  ctaLink: string;
+  endingTimeMs?: number;
+};
+export const Banner: React.FC<BannerProps> = (props) => {
+  const { maxWidth, iconSrc, titleKey, descriptionKey, ctaKey, ctaLink, endingTimeMs } = props
```

Comment:
`ctaKey` isn't consumed anywhere

Reply:
totallly, it felt necessary, i don't think it is though since we have other fields that serve the purpose. removing.